src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -93,10 +93,30 @@ are ignored by exporter. Duplicate keys are exported as is.
 
 You can configure the `OtlpExporter` through `OtlpExporterOptions`
 and environment variables.
-The `OtlpExporterOptions` type setters take precedence over the environment variables.
 
-This can be achieved by providing an `Action<OtlpExporterOptions>` delegate to the
-`AddOtlpExporter()` method or using `AddOptions<OtlpExporterOptions>()`.
+> **Note**
+> The `OtlpExporterOptions` type setters take precedence over the environment variables.
+
+This can be achieved by providing an `Action<OtlpExporterOptions>` delegate to
+the `AddOtlpExporter()` method or using the `Configure<OtlpExporterOptions>()`
+Options API extension.
+
+> **Note**
+> The `OtlpExporterOptions` class is shared by logging, metrics, and tracing. To
+> specify different settings for each signal use the `name` parameter on the
+> `AddOtlpExporter` extensions:
+>
+> ```csharp
+> appBuilder.Services.AddOpenTelemetry()
+>     .WithTracing(builder => builder.AddOtlpExporter("tracing", configure: null))
```

Comment:
Maybe provide a non-null configure Action for one of the signals. This might give the wrong impression to people who are new that this the right way to do it: 
1. Provide a null configure Action.
2. Then,use appBuilder.Services.Configure<OtlpExporterOptions>

Reply:
I just switched it to be specific to binding the options via `IConfiguration`. If you supply the delegate inline to do code-based configuration you really don't need to use the name at all so I thought this would be better LMK.